zingolib/src/lightclient/describe.rs

```diff
@@ -20,15 +20,25 @@ use crate::{
     error::ZingoLibError,
     wallet::{
         data::{
-            finsight, summaries::ValueTransfer, summaries::ValueTransferKind, OutgoingTxData,
-            TransactionRecord,
+            finsight,
+            summaries::{ValueTransfer, ValueTransferKind},
+            OutgoingTxData, TransactionRecord,
         },
         keys::address_from_pubkeyhash,
         notes::{query::OutputQuery, OutputInterface},
+        transaction_records_by_id::TransactionRecordsById,
         LightWallet,
     },
 };
 
+#[allow(missing_docs)]
+#[derive(Debug, thiserror::Error)]
+pub enum ValueTransferRecordingError {
+    #[error("Fee was not calculable because of error:  {0}")]
+    FeeCalculationError(String),
```

Comment:
rather than strings would we benefit from holding the actual error type? this is how I have done error stacks and have seen in other codebases. not sure what the pros and cons are though but I think we only lose information or potential error handling improvements by converting to string here

Reply:
I will look into this.   My guess is that it's OK to lose type information *ONLY* if you're about to uniquely consume the error.

Reply:
Unless this is blocking I am leaving it with a TODO note.